src/DonorProfiles/resources/js/app/tabs/annual-receipts/content.js

```diff
@@ -1,12 +1,34 @@
+import { Fragment, useEffect } from 'react';
+
+const { __ } = wp.i18n;
+
 import Heading from '../../components/heading';
-import { Fragment } from 'react';
+import AnnualReceiptTable from '../../components/annual-receipt-table';
+
+import { useSelector } from './hooks';
+import { fetchAnnualReceiptsFromAPI } from './utils';
 
 const Content = () => {
-	return (
+	const annualReceipts = useSelector( ( state ) => state.annualReceipts );
+	const querying = useSelector( ( state ) => state.querying );
+
+	useEffect( () => {
+		fetchAnnualReceiptsFromAPI();
+	}, [] );
+
+	return querying === true && annualReceipts === null ? (
+		<Fragment>
+			<Heading>
+				{ __( 'Loading...', 'give' ) }
+			</Heading>
+			<AnnualReceiptTable />
+		</Fragment>
+	) : (
 		<Fragment>
 			<Heading>
-				Annual Receipts
+				{ annualReceipts ? `${ Object.entries( annualReceipts ).length } ${ __( 'Total Annual Receipts', 'give' ) }` : __( '0 Total Annual Receipts', 'give' ) }
```

Comment:
I would suggest using a single translatable string and use `sprintf` to insert the number.

Reply:
Hmm -- without adding another JS library to support a formatter like sprintf, I've opted to abstract the annual receipts count determination logic into it's own line during the render method. This ensures that only 1 translation is needed. Ref: https://github.com/impress-org/givewp/pull/5611/commits/6b4315f1bf401bf8712084a2cce674bd44ff3e82